### DISCUSSION TITLE
Cleaning up after installing the development version from the repository

### DISCUSSION THREAD

**Discussion by snhansen** [quarto-dev/quarto-cli#11313] [Q&A]
### Description

While waiting for a new pre-release build to be available, I decided to try building the newest development version directly from the repository as I was needing some of the fixes which isn't yet available in the newest pre-release build. I ran into some issues and decided to delete the install and simply wait for the pre-release build to become available.

However, when I now run `quarto preview` I get an "Internal Server Error" in my browser and the CLI shows

```
Browse at http://localhost:3820/filename/test-123.html
ERROR: 500 (Internal Error): The extension Poster is incompatible with this quarto version.

Extension requires: >=99.9.0
Quarto version: 1.6.32
```

The rendering works fine, but it's the browser preview that fails. Since it says `Extension requires: >=99.9.0` I take it that I have some leftovers from the repo-build and I'm wondering how I can clean up after this?

**Comment by mcanouil** [comment]
What's the "Poster" extension? (I am not aware of it https://m.canouil.dev/quarto-extensions/)

Because this error/warning is usually when `quarto-required: ">=99.9.0"` is set in the extension.
The Quarto version is the version of Quarto

**Reply by snhansen** [reply]
No idea. It's not something I've installed nor is it something that's in the project directory. The above is a simple "Hello World" `.qmd` with `format: html`.

**Reply by mcanouil** [reply]
Maybe https://github.com/quarto-ext/typst-templates/tree/main/poster

Are you sure you did not set `quarto-required` somewhere? Document metadata header? Project `_quarto.yml`?

Does the error appear if you create in another location a new project such as `quarto create project website`?

**Comment by snhansen** [comment]
I just tried rendering from a different location which worked, thanks! 

Turns out that in the parent directory of the project directory was a `.quarto` folder which I deleted and next time I rendered it was recreated and the issue was resolved.

**Reply by cscheid** [reply]
Thanks for the additional info. This feels like a Quarto bug.

Do you also have a `_quarto.yml` file on the folder that has the `.quarto` directory?

**Reply by snhansen** [reply]
Yes, there was a `_quarto.yml` in that folder with the following content:

```
format: 
  revealjs:
    slide-number: true
    smaller: true
    width: 1100
    chalkboard:
      theme: whiteboard
```

It wasn't meant to have anything to do with the project folder in which the error occurred, but was a left over from some previous tests. It might have interfered though.
